index.js

```diff
@@ -2,14 +2,14 @@
 
 'use strict';
 
-if (hexo.config.nofollow && hexo.config.nofollow.enable) {
-  const config = hexo.config.nofollow;
+const config = Object.assign({
+  enable: true
+}, hexo.config.nofollow);
 
-  if (!config.field) config.field = 'site';
+if (!config.field) config.field = 'site';
 
-  if (config.field === 'post') {
-    hexo.extend.filter.register('after_post_render', require('./lib/filter'));
-  } else {
-    hexo.extend.filter.register('after_render:html', require('./lib/filter'));
-  }
+if (config.field === 'post') {
```

Comment:
should have `if (config.enable === true)` before this line.

Reply:
Forgot it. `if (!config.enable) return;` is now added.